hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -310,6 +310,19 @@ protected synchronized boolean areSomeRegionReplicasColocated(Cluster c) {
     return false;
   }
 
+  private synchronized boolean areThereIdleRegions(Cluster c){
```

Comment:
How about putting this method in `BaseLoadBalancer` as `protected final` ?

Reply:
Also, `idleRegionsExist()` might be better name

Reply:
Here it checks the status of RS.  The name areThereIdleRegions not making good meaning

Reply:
Yeah, it could be something similar to idleRSPresent() maybe

Reply:
thanks!